lib/rubocop/cop/minitest/any_assertions.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This cop checks if test cases contain any assertion calls.
+      #
+      # @example
+      #   # bad
+      #   class FooTest < Minitest::Test
+      #     def test_the_truth
+      #     end
+      #   end
+      #
+      #   # good
+      #   class FooTest < Minitest::Test
+      #     def test_the_truth
+      #       assert true
+      #     end
+      #   end
+      #
+      class AnyAssertions < Cop
```

Comment:
Oh, I seem to have forgotten to comment 💦 

I think the cop name `NoAssertionMethods` is clear for registering offenses.

```suggestion
      class NoAssertionMethods < Cop
```

Reply:
How about `NoAssertions`, to match `MultipleAssertions` ?

Reply:
`NoAssertions` also looks good name. And I think `MultipleAssertions` will be different role :-)

Reply:
Okay, I can change it tomorrow. But I’m also fine with it now if you want to merge.

Reply:
Thanks. If you like, can you replace it with new `Base` API for the cop?

```diff
diff --git a/lib/rubocop/cop/minitest/no_assertion_methods.rb b/lib/rubocop/cop/minitest/no_assertion_methods.rb
index 2292d03..36671ee 100644
--- a/lib/rubocop/cop/minitest/no_assertion_methods.rb
+++ b/lib/rubocop/cop/minitest/no_assertion_methods.rb
@@ -19,7 +19,7 @@ module RuboCop
       #     end
       #   end
       #
-      class NoAssertionMethods < Cop
+      class NoAssertionMethods < Base
         include MinitestExplorationHelpers

         MSG = 'Test case has no assertions.'
@@ -32,7 +32,7 @@ module RuboCop

             next if assertions_count.positive?

-            add_offense(node, location: node.block_type? ? :expression : :name)
+            add_offense(node.block_type? ? node.loc.expression : node.loc.name)
           end
         end
```

e.g.: https://github.com/rubocop/rubocop-minitest/pull/128

Reply:
@koic updated!